app/controllers/api_controller/parser/request_adapter.rb

```diff
@@ -39,15 +58,7 @@ def method
       end
 
       def path
-        URI.parse(url).path.sub(/\/*$/, '') # /api/...
-      end
-
-      def subcollection
-        @params[:subcollection]
-      end
-
-      def s_id
-        @params[:s_id]
+        URI.parse(url).path.sub(%r{\/*$}, '') # /api/...
```

Comment:
I believe that if you're converting to `%r{}` you can remove the escaping of `/` here